recipes/pyilluminate/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pyilluminate" %}
+{% set version = "0.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # I can't seem to download from pypi....
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # sha256: c94721144ee4388175edaa31f14388da3101b4583a3368f047fdcf7797da1a4d
+  url: https://github.com/ramonaoptics/pyilluminate/archive/{{ version }}.tar.gz
+  sha256: 7e30a85aa1c3d49e6f59f053f2f969cc77b2505c764dcf244d52bd65878195f8
```

This should work when the name is changed:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: c94721144ee4388175edaa31f14388da3101b4583a3368f047fdcf7797da1a4d
```

---

Whoa, thanks, I'm just going to change the name to lowercase on pypi ;). Lets see how badly I'm going to break things.